src/Mod/TechDraw/Gui/Command.cpp

```diff
@@ -287,7 +287,9 @@ CmdTechDrawView::CmdTechDrawView() : Command("TechDraw_View")
     sAppModule = "TechDraw";
     sGroup = QT_TR_NOOP("TechDraw");
     sMenuText = QT_TR_NOOP("Insert View");
-    sToolTipText = QT_TR_NOOP("Insert a View");
+    sToolTipText = QT_TR_NOOP("Insert a View in current page.\n"
```

Comment:
Insert a view in current page.
Selected objects, spreadsheets or Arch WB section planes will be added with the current camera orientation.
Without a selection, a file browser lets you select a SVG or image file.